app/models/post.rb

```diff
@@ -5,16 +5,19 @@
 #  id         :integer          not null, primary key
 #  author     :string           not null
```

Comment:
not something you need to do in this PR, but we should remove the author field cuz user_id takes care of that